tests/web/test_servicex_file.py

```diff
@@ -36,3 +38,47 @@ def test_servicex_file_ambiguous_match(self, client):
         flashed_messages = get_flashed_messages()
         assert flashed_messages
         assert "Unable to infer filetype" in flashed_messages[0]
+
+    def test_correct_url(self, client):
```

Comment:
This is a perfect use case for a [parameterized test](https://docs.pytest.org/en/stable/parametrize.html).

```
@pytest.mark.parametrize("test_input,expected", [("3+5", 8), ("2+4", 6), ("6*9", 42)])
def test_eval(test_input, expected):
    assert eval(test_input) == expected
```